Library/Invoice.cs

```diff
@@ -665,12 +683,18 @@ internal void WriteUpdateXml(XmlTextWriter xmlWriter)
         {
             xmlWriter.WriteStartElement("invoice"); // Start: invoice
 
-            Address.WriteXml(xmlWriter);
-            xmlWriter.WriteElementString("customer_notes", CustomerNotes);
-            xmlWriter.WriteElementString("terms_and_conditions", TermsAndConditions);
-            xmlWriter.WriteElementString("vat_reverse_charge_notes", VatReverseChargeNotes);
-            xmlWriter.WriteElementString("gateway_code", GatewayCode);
-            xmlWriter.WriteElementString("po_number", PoNumber);
+            if (!Address.Equals(_referenceAddress)) Address.WriteXml(xmlWriter);
+
+            if (CustomerNotes != null)
+                xmlWriter.WriteElementString("customer_notes", CustomerNotes);
```

Comment:
This will technically be a breaking change, but I think it will add value along the same lines as the conversation here https://github.com/recurly/recurly-client-net/pull/368#discussion_r246890848

We'll go ahead and bump the minor version when we release this.